app/src/pages/CounterPage/containers/CounterPageContainer.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react';
+import Counter from '../components/Counter';
+import PropTypes from 'prop-types';
+
+class CounterPageContainer extends Component {
+    constructor(props) {
+
+        super(props);
+        this.state ={
+            countValue:0,
+            parity:'even',
+            countScreen:'countScreenR'
+
+
+
+        }
+    }
+    handleIncrement =() =>{
+    this.setState((state)=>{
+        const countValue = state.countValue+1;
+        let parity= ''
+        let countScreen =''
```

Comment:
why are you using `let` you can use `const` and decide approriate value with ternary operators